src/scripts/content.js

```diff
@@ -56,14 +56,19 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
         let message = "Logged out";
         const krHudElement = document.getElementsByClassName('mousehuntHud-huntersHorn-response')[0];
         const krPageElement = document.getElementsByClassName('mousehuntPage-puzzle-form-state hasPuzzle')[0];
-        const hunt_timer = document.getElementById('huntTimer');
+        const hunt_timer = document.querySelector(".huntersHornView__countdown");
+        const is_ready = document.querySelector('.huntersHornView__timerState--type-ready');
```

Comment:
I think we can just confirm the hunt_timer hasClass('ready')

Reply:
Updated in `7993742bcb21` 👍